appcenter-data/__tests__/Data.js

```diff
@@ -77,12 +77,13 @@ describe('App Center Data read operation tests', () => {
 });
 
 describe('App Center Data list operation tests', () => {
-  test('list is called ', async () => {
+  test('list is called with partition and TimeToLive.DEFAULT', async () => {
     const partition = Data.DefaultPartitions.USER_DOCUMENTS;
     const spy = jest.spyOn(NativeModules.AppCenterReactNativeData, 'list');
     const result = await Data.list(partition);
     console.log(result);
-    expect(spy).toHaveBeenCalledWith(partition);
+    const readOptions = { "timeToLive": Data.TimeToLive.DEFAULT };
```

Comment:
undefined should map to default, we are supposed to have code to handle the default case
if you had to update the unit test then it might hide a real bug in the SDK

Reply:
@guperrot `await Data.list(partition)` is called without TTL parameter (line 82) and the SDK correctly treats it `TimeToLive.DEFAULT`. The problem was this test case was failing on develop branch but unit tests of Data module and Auth module are not executed in the CI. So this PR fixed a bug in the unit test, and now all the JS unit tests are executed in the CI.